lib/staccato.ex

```diff
@@ -10,18 +10,14 @@ defmodule Staccato do
 
       iex> Staccato.tracker("X-YYYYY-1").id
       "X-YYYYY-1"
+  """
+  def tracker(id, client_id \\ UUID.uuid4(), options \\ [])
 
-      iex> Staccato.tracker(nil)
-      %Staccato.NoopTracker{}
+  def tracker(nil, _, _), do: raise("Property ID is required")
```

Comment:
I'd like to keep the `NoopTracker`. It's not really useful just yet, but will be very soon.

Reply:
Gotcha, couldn't tell your intentions here but am happy to revert.

Reply:
Yeah, it makes certain testing a lot easier when using this library in other projects.